### PR TITLE
Add thee months out blog post

### PR DESCRIPTION
**Pull request by ericholscher**
This will go live tomorrow.

----
📚 Documentation preview 📚: https://writethedocs-www--2079.org.readthedocs.build/conf/portland/2024/news/three-months-out/

### CODE REVIEW

File: docs/conf/portland/2024/news/three-months-out.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+template: {{year}}/generic.html
+banner: _static/conf/images/headers/portland-2024-speakers.jpg
+og:image: _static/conf/images/headers/portland-2024-speakers.jpg
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/three-months-out.md:4]: Is this a default already and this is just an override?

File: docs/conf/portland/2024/news/three-months-out.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+.. post:: Jan 18, 2024
+   :tags: {{shortcode}}-{{year}}, tickets, guide, visiting
+```
+
+
+# Write the Docs: Only 3 Months Away!
+
+Our upcoming conference is right around the corner, and we are excited to reunite with everyone in Portland. Now is a great time to buy your tickets and plan travel arrangements.
+
+Explore more information below on the attendee guide, visiting Portland, strategies for convincing your manager, deadlines for the opportunity grant, and exciting upcoming speaker announcements.
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/three-months-out.md]:
```suggestion
Read on for more information on the attendee guide, visiting Portland, strategies for convincing your manager, deadlines for the opportunity grant, and exciting upcoming speaker announcements.
```

Comment (plaindocs) [reply docs/conf/portland/2024/news/three-months-out.md]: Could link these so folks can skip, but :man_shrugging:

File: docs/conf/portland/2024/news/three-months-out.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+- Student or Unemployed Tickets: $100
+- Independent Tickets: $350
+- Corporate Tickets: $650
+
+[**Buy your ticket today!**](https://www.writethedocs.org/conf/portland/2024/tickets/)
+
+## Attendee Guide and Visiting Portland
+
+Our [Attendee Guide](https://www.writethedocs.org/conf/portland/2024/attendee-guide/) and [Visiting Portland](https://www.writethedocs.org/conf/portland/2024/visiting/) pages are now live! We've gathered information that will help you navigate the conference like a pro and make you feel more at home in the city of Portland.
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/three-months-out.md]:
```suggestion
Our [Attendee Guide](https://www.writethedocs.org/conf/portland/2024/attendee-guide/) and [Visiting Portland](https://www.writethedocs.org/conf/portland/2024/visiting/) pages are now live! We've gathered information that will help you navigate the conference comfortably and make you feel more at home in the city of Portland.
```

File: docs/conf/portland/2024/news/three-months-out.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+The [Attendee Guide](https://www.writethedocs.org/conf/portland/2024/attendee-guide/) provides conference information and gives strategies and tips for new (and experienced) attendees to get plugged in and connect with others.
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/three-months-out.md:35]: Could probably use a rewrite but I don't have much right now

File: docs/conf/portland/2024/news/three-months-out.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+The [Visiting Portland](https://www.writethedocs.org/conf/portland/2024/visiting/) has info about where to stay, eat, drink, and how to get around the city!
+
+![Group of people talking](/_static/img/2024/attendee-guide.jpg)
+
+## Convince Your Manager
+
+Do you need help justifying why your employer should send you to Write the Docs? Don't worry -- you're not alone. Based on the experiences of some of our previous attendees, we've put together a sample email and list of resources.
+
+[**Convince your manager.**](https://www.writethedocs.org/conf/portland/2024/convince-your-manager/)
```
Comment (plaindocs) [inline docs/conf/portland/2024/news/three-months-out.md]:
```suggestion
[**Convince your manager**](https://www.writethedocs.org/conf/portland/2024/convince-your-manager/)
```

**Review by plaindocs — APPROVED** [review]
Optional nits throughout.

File: docs/conf/portland/2024/news/three-months-out.md (hunk 1 above)
Comment (ericholscher) [reply docs/conf/portland/2024/news/three-months-out.md:4]: I couldn't find a good way to deafult it.

File: docs/conf/portland/2024/news/three-months-out.md (hunk 4 above)
Comment (ericholscher) [reply docs/conf/portland/2024/news/three-months-out.md:35]: Aye, I did some small edits, but didn't futz with it too much.

File: docs/conf/portland/2024/news/three-months-out.md (hunk 1 above)
Comment (plaindocs) [reply docs/conf/portland/2024/news/three-months-out.md:4]: Ah, sad times.

**Pull request merged**